### DISCUSSION TITLE
Excited about the New Course

### DISCUSSION THREAD

**Discussion by rohitkalpande** [giswqs/geog-312#11] [General]
I just came across the LinkedIn post for this Course, I was astonished. The art of curation at it best, to use open source for maximum benefits of students and providing value by enriching knowledge. I am so grateful for it sir. Lastly Thank you so much.

**Comment by giswqs** [comment]
Thank you for tuning in

**Comment by Victorsanval1** [comment]
Hi, new in the course from Spain!! I'm a student in telecomunication studies and working for a spacial company. 
Thanks por the open sources, great job. 

Wanted to ask professor Wu , if login in the canvas is needed for following the course. And if at the end a certification is given. 
Thanks in advance

**Reply by giswqs** [reply]
No canvas login is needed. No certificate either.
